.github/workflows/test-s3gw.yml

```diff
@@ -156,24 +121,44 @@ jobs:
 
           docker kill $CONTAINER
 
-      - name: Run S3tests
+      - name: Run S3 Tests
+        run: |
+          set -x
+
+          docker run --rm \
+            -v /run/docker.sock:/run/docker.sock \
+            -v ${GITHUB_WORKSPACE}/s3tr-out:/out \
+            ghcr.io/aquarist-labs/s3tr:latest \
+            run \
+            --image s3gw:${IMAGE_TAG} \
+            --docker-api unix:///run/docker.sock \
+            --nproc 10 \
+            /out/s3tr.json
+
+      - name: Archive Results
+        uses: actions/upload-artifact@v3
+        with:
+          name: s3tests-results-${{ github.workflow_sha }}-${{ github.run_id }}-${{ github.run_attempt }}
+          path: s3tr-out/s3tr.json
+
+      - name: Analyze Results
         run: |
           set -x
 
-          # needed for GNU parallel because the version in GH runners is bugged
-          mkdir -p ${HOME}/.parallel
-
-          export DEBUG=1
-          export CEPH_DIR="${GITHUB_WORKSPACE}/ceph"
-          export OUTPUT_DIR="${GITHUB_WORKSPACE}/s3test-results"
-          export S3GW_CONTAINER="s3gw"
-          export FORCE_CONTAINER=ON
-          export FORCE_DOCKER=ON
-          export \
-            FIXTURES="${GITHUB_WORKSPACE}/ceph/qa/rgw/store/sfs/tests/fixtures"
-
-          export S3TEST_REPO="${GITHUB_WORKSPACE}/s3tests"
-          export S3TEST_CONF="${FIXTURES}/s3tests.conf"
-          export S3TEST_LIST="${FIXTURES}/s3-tests.txt"
-          pushd s3tests
-          ${GITHUB_WORKSPACE}/s3gw/tools/tests/s3tests-runner.sh
+          docker run --rm \
+            -v ${GITHUB_WORKSPACE}/s3tr-out:/out \
+            -v ${GITHUB_WORKSPACE}/ceph:/ceph:ro \
+            ghcr.io/aquarist-labs/s3tr:latest \
+            analyze new-successes \
+              --known-good-file \
+                /ceph/qa/rgw/store/sfs/tests/fixtures/s3-tests.txt \
+              /out/s3tr.json
+
+          docker run --rm \
+            -v ${GITHUB_WORKSPACE}/s3tr-out:/out \
+            -v ${GITHUB_WORKSPACE}/ceph:/ceph:ro \
+            ghcr.io/aquarist-labs/s3tr:latest \
+            analyze new-failures \
+              --known-good-file \
+                /ceph/qa/rgw/store/sfs/tests/fixtures/s3-tests.txt \
+              /out/s3tr.json
```

Comment:
It might be a good idea to put an `echo` or something here that helps with interpretation of the output.

Reply:
Agree, output is not optimal. 
There'll be a rewrite of the success/failure logic to "oktofail.csv" - Nicer output will be part of that.